### PR TITLE
add ergonomics intro

### PR DESCRIPTION
**Pull request by mcraa**
- [x] proofread
- [x] i tried to use the image in the header as "/blog/folder/image.jpg" but it did not load locally is the "featuredImage" property working with full URLs only?
- [x] added the public folder to .gitignore , any reason it was not there before?

### CODE REVIEW

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+That won't tell you much right? It is more like a guideline for me to speak about the same thing from different perspectives for 30 minutes. Let me summarize it here:
+
+#### See the squatting kid on the first slide?
+
+Can you do the same? Are you sure? Double check your ankles, are the falling in or they are correctly in lines with your knees? Or your knees are falling inwards too? Are your heels on the ground (barefoot, not in raised heel shoes)? Is your back straigt? Can you look up easily?
+
+If any of the questions are not a unquestionable yes you are prone to have some problems from the office-desk environment.
+
+We were born with the ability to be like the kid on the image. And we slowly learned to lose that. By sitting in scool, by sittin in cars, on the bus, at office desks, etc. Your body is becoming better in things you practice. You practiced to develop back pain for years.
```
Comment (marcusstigelid) [inline content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:41]: Spelling: "...like the kid on the image." -> "...like the kid **in** the image."
Spelling: "scool" -> "school", "sittin" -> "sitting"

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+> _Header <a href="https://www.freepik.com/free-vector/workplace-stretches-cartoo-icons-set-with-people-having-exercises-office-isolated-vector-illustration_37366242.htm#query=office%20stretching&position=30&from_view=keyword&track=ais&uuid=be3dc103-793b-4e3a-85e2-2b2931f5778a">Image by macrovector</a> on Freepik_
+
+### We love to share our knowledge and improve each other
+
+That is why we have this series of presentation about office ergonomics we also want to share with the public. Yes series, this is just the beginning a general introduction to the topic. Later we will have more specific examples.
+
+### Who are you to tell me what to do
+
+I am not telling you what to do but recomend ways to keep your joints and bones and muscles healthy, based on many researches and my own experience. I have been both a practicioner and a trainer in many different disciplines including
```
Comment (marcusstigelid) [inline content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:20]: Spelling: "recomend" -> "recommend", -> "practicioner" -> "practitioner"
Wording: "based on many researches" -> "based on a lot of research"

**Comment by marcusstigelid** [comment]
As you can see I added some comments, mainly regarding wordings and spellings. Otherwise this looks great! Nice to see that the blog gets updated again.

**Comment by marcusstigelid** [comment]
Regarding the public folder I am not sure why it hasn't been added before. I only noticed it now as well that a lot of files were generated and that we didn't ignore them, so it's great that you've added it.

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+If any of the questions are not a unquestionable yes you are prone to have some problems from the office-desk environment.
+
+We were born with the ability to be like the kid in the image. And we slowly learned to lose that. By sitting in school, by sitting in cars, on the bus, at office desks, etc. Your body is becoming better in things you practice. You practiced to develop back pain for years.
+
+#### If you haven't peaked to the 4th slide yet
+It is the time now. As I said "for years" just above, it is clear what we are trying to fight here is not happening quickly but steadily. We need to get rid of the bad habits, namely being stationary in one position, getting too comfortable in a position so we are looking forward to get back into that position ant that only telling our body to morph into that shape.
+
+Having a little back pain may be the most "visible" effect of a bad posture but you won't guess all the side effect even in your worst dreams of a little pain in an allegedly insignificant body part.
```
Comment (Dunklas) [inline content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:46]: I had some trouble reading this sentence as well - My tongue twists a bit after "in your worst dreams". Is the "of a little pain in an allegedly insignificant body part"-part necessary? If so, maybe you could rephrase or split up the sentence?

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+> Sidenote here a book recommendation [The Align Method by Aaron Alexander](https://www.google.se/books/edition/The_Align_Method/WnmRDwAAQBAJ?hl=en&gbpv=1&printsec=frontcover) Where the author explains how the connection between mood and body posture a two way street. Meaning when you are sad for example you tend to curl up. But if you curl up your body will interpolate the position as a sign to be sad. Standing tall makes you happier, more confident, not just healthyer in the posture.
```
Comment (Dunklas) [inline content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:48]: healthyer -> healthier

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+Simple example the already mentioned inward falling ankle during a squat makes your knee compensate which makes your hip, compensate which makes your core muscles. Everything is connected. Keep your house of cards stable.
```
Comment (Dunklas) [inline content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:50]: "_Simple example the already mentioned inward falling ankle during a squat makes your knee compensate which makes your hip, compensate which makes your core muscles._"

I think you should move and introduce commas here. Something like: "makes your knee compensate, which makes your hip compensate, which makes your core muscles compensate."

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+#### Now you reached the DEMO slide
+Sorry this is something I can't put in text, with a volunteer we demonstrated how seemingly distant things in your body can effect each other, and went trough some recommendations of standing desk setups.
+
+If you want to see this part as well, [drop us a message and become a devie](https://www.devies.se/become-a-devie/)
+
+#### Don't take the joke slides below the solution too seriously
+
+But to keep it real what did we learn? There is no one chair there is no one magic position standing, or classic desk which will make your posture better. Your enemy is not moving. Locking out your joints in a stationary position. Resting in that position for a long time. And "suddenly" you realize you are stuck there forever.
+
+The key is to variate your positions. Either in one place or moving to different stations. If you read the recommended book you may go to the ground too. Don't be affraid to try new things.
```
Comment (Dunklas) [inline content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:63]: affraid -> afraid

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+- Get a small water bottle, when it gets empty get up and walk to a possibly quite far kitchen.
+
+On the next slides you see some stock images about more or less useful ergonomic solutions you see all of them are non stationary.
+Then you can do some streching directly at your desk or have a separate section in your office to some "offce yoga". Of course it is very much recommended to do similar activities outside the office too.
+
+#### Follow up
+
+We will continue with 
+ - more specific examples what problems people have 
+ - and what excercises can we do in the office
```
Comment (Dunklas) [inline content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:77]: excercises -> exercises

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+slug: "the-office-movement-aka.-how-to-stay-healthy-in-the-office"
+author:
+ - Péter Makra
+---
+
+> _Header <a href="https://www.freepik.com/free-vector/workplace-stretches-cartoo-icons-set-with-people-having-exercises-office-isolated-vector-illustration_37366242.htm#query=office%20stretching&position=30&from_view=keyword&track=ais&uuid=be3dc103-793b-4e3a-85e2-2b2931f5778a">Image by macrovector</a> on Freepik_
+
+### We love to share our knowledge and improve each other
+
+That is why we have this series of presentation about office ergonomics we also want to share with the public. Yes series, this is just the beginning a general introduction to the topic. Later we will have more specific examples.
```
Comment (Dunklas) [inline content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:16]: I think it'd be nice to explain that you recently held a talk on this topic, and that the purpose of this post is too summarize key points or something :) Makes it a bit feel a bit more natural with the slides, etc.

**Review by Dunklas — COMMENTED** [review]
Sorry about the late review! Great post, and I love that you're bringing this topic into the organization! :)

I've left a few comments, both on spelling and regarding some sentences which I had a bit trouble reading.

And a tip for coming posts is to write one sentence per line.
It'll look the same when published, but is awesome while writing and reviewing :)
I started to do it after I stumbled over [this post](https://sive.rs/1s) and have made it my default behavior everytime I write markdown :)

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md (hunk 3 above)
Comment (mcraa) [reply content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:46]: We need to get rid of the bad habits.
Focusing now on the habit of being stationary in one position.
Getting too comfortable in a position, so we are looking forward to getting back into that same position.
Which only tells our body to slowly morph into that shape.

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md (hunk 5 above)
Comment (mcraa) [reply content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:50]: Simple example the already mentioned inward falling ankle during a squat. Which makes your knee compensate, which makes your hip compensate, which makes your core muscles uneven.

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md (hunk 8 above)
Comment (mcraa) [reply content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:16]: That is why we have this series of presentations in the office about office ergonomics. 
Yes series, this is just the beginning, a general introduction to the topic.
So this is a summary of the initial presentation.
Later we will have more specific examples.

File: content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+#### See the squatting kid on the first slide?
+
+Can you do the same? Are you sure? Double check your ankles, are the falling in or they are correctly in lines with your knees? Or your knees are falling inwards too? Are your heels on the ground (barefoot, not in raised heel shoes)? Is your back straigt? Can you look up easily?
+
+If any of the questions are not a unquestionable yes you are prone to have some problems from the office-desk environment.
+
+We were born with the ability to be like the kid in the image. And we slowly learned to lose that. By sitting in school, by sitting in cars, on the bus, at office desks, etc. Your body is becoming better in things you practice. You practiced to develop back pain for years.
+
+#### If you haven't peaked to the 4th slide yet
+It is the time now. As I said "for years" just above, it is clear what we are trying to fight here is not happening quickly but steadily. We need to get rid of the bad habits, namely being stationary in one position, getting too comfortable in a position so we are looking forward to get back into that position ant that only telling our body to morph into that shape.
```
[the remark this answers is not in the document]
Comment (mcraa) [reply content/posts/2024-03-15_the-office-movement-aka.-how-to-stay-healthy-in-the-office.md:44]: We need to get rid of the bad habits.
Focusing now on the habit of being stationary in one position.
Getting too comfortable in a position, so we are looking forward to getting back into that exact same position.
Which only tells our body to slowly morph into that shape.

**Comment by mcraa** [comment]
@marcusstigelid @Dunklas 
I updated everything mentioned.
Good points!

**Review by Dunklas — APPROVED** [review]
Awesome! Regarding the featured image - For me it works OK when setting the featured_image property to "/ergonomics/header_streches.jpg".
Could you try that?

**Comment by mcraa** [comment]
> Awesome! Regarding the featured image - For me it works OK when setting the featured_image property to "/ergonomics/header_streches.jpg". Could you try that?

💪 probably the problem was that I included the "/blog" prefix in the path.

**Review by Dunklas — APPROVED** [review]

**Pull request merged**